dask_jobqueue/core.py

```diff
@@ -153,8 +154,8 @@ def start_workers(self, n=1):
         workers = []
         for _ in range(n):
             with self.job_file() as fn:
-                out = self._call([self.submit_command, fn])
-                job = out.decode().split('.')[0]
+                out = self._call(shlex.split(self.submit_command) + [fn])
```

Comment:
I don't know shlex, is this mandatory? It serves what purpose?

Reply:
In SGE, by default qsub returns quite a verbose output, e.g. something like `Your job 56 ("test.sh") has been submitted`. In order for just the job id to be returned, you need to use `qsub -terse`. This is why `submit_cmd = 'qsub -terse'`.

That means that you need to split `submit_cmd`. I think `shlex.split` is the way to do it for sh commands. We could just do `str.split but it may break e.g. if one of the arguments is quoted with a space inside.